README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## This is an autobiography from Yinsu Han for HW01.
+My name is **Yinsu Han**. I am a rising sophomore in the college, planning to major in **mathematics**. I am always excited to learn new things. Here are several things I enjoy apart from learning:
+
+* *Playing basketball*
+* *Listening to music/Going to concerts*
+* *Hanging out with my friends*
+
+### This is a picture of mine in downtown Chicago
+![picture](https://github.com/YinsuH/hw01/blob/master/dt_chicago.JPG)
```

Comment:
Nice embedding of the image! You can also just use `![](dt_chicago.JPG)` since `README.md` is in the same folder as the picture

Reply:
from your own reflection and the number of commits, I see you also had difficulty with figuring out how to upload a picture! it also took me a while and a bunch of trial and error to get it work as well-I agree that was probably the most difficult aspect of this hw! I didn't realize you could use links to get them to show up either!